cpp/ql/lib/semmle/code/cpp/dataflow/internal/DataFlowPrivate.qll

```diff
@@ -114,13 +114,20 @@ abstract class OutNode extends Node {
   abstract DataFlowCall getCall();
 }
 
+/**
+ * Represents a private class that extends OutNode and ExprNode.
+ * This class is used for expressing data flow analysis in C++ code.
+ */
```

Comment:
The two comments in this file are garbage.  What I'd like is something like the doc for `OutNode` (above), preferably with examples because it isn't obvious what is and is not considered the "output" of a call.